CHANGELOG.md

```diff
@@ -20,13 +20,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.38.0] - 2023-04-04
 ### Added
 - New feature to allow for single component testing as code is refactored
-- Created `Comp_Testing_Driver.F90` in Apps
-- Created `subset_fields.py` to grab a subset of columns
-- Created shell script to perform subsetting, run the driver, and evaluate differences
-
-### Changed
-- Changed `MAPL_Generic.F90` to capture states before and after run and perform export restart if requested
-- Changed `NCIO.F90` to allow for capture of a subset of columns and perform export restart if requested
+  - Created `Comp_Testing_Driver.F90` in Apps
```

Comment:
I wonder if we could/should put a link to the wiki instructions here?

Reply:
Are you saying you think I should make a new MAPL wiki page from `Comp_Testing_Info.md`  and link it here?

Reply:
Ah sorry.   Just was not thinking clearly.   You have the instructions above as a markdown page.    And I'd thought of it as a wiki by the time I got down here.   So just give the path to the markdown file.   A bit risky in that we may move the file later on but will never change the log.   But while it is a new feature, it helps people to know how to go use it if they are interested.